wire-ios-data-model/Source/Model/Conversation/ZMConversation+Creation.swift

```diff
@@ -171,34 +171,20 @@ extension ZMConversation {
         return conversation
     }
 
-    /// FOR TESTS ONLY.
-    /// To create new conversations see ConversationService.
-
-    @objc
-    static func fetchOrCreateOneToOneTeamConversation(
+    static func fetchOneToOneTeamConversation(
```

Comment:
I'm not sure what do you mean, because in this file all the methods have different formats. Without a consistent style guide that all team members follow, I feel like I'll have to guess.

Reply:
My mistake I thought it already had consistent formatting. It's a fair point and perhaps we should bring up the topic of a style guide in the next collective.